### ISSUE TITLE
FANTASMA: phantom recipient

### ISSUE THREAD

**Issue by evilaliv3** [globaleaks/GlobaLeaks#1728]
Working on the BustiaEtica project we collected a possible feature involving a FANTASMA.

The keypoints discussed for such a feature are:
- the platform COULD allow to define a phantom recipient
- the platform, upon creation of the Phantom recipient MUST leave on the platform only the public key of the phantom user and force the administrator to download the fantasma key.
- it will be duty of the adopter manage the safe archival and the authorization process for accessing the phantasma key in priority situations (like big data losses on recipient sides / corrupted recipients)

**Comment by evilaliv3** [comment]
By current research ideas, i think that this phantasma user could be eventually the same used for implementing the key escrow mechanism.

In my opinion in fact the two features does not compete one on the other; both enable different use cases and the implementation of both add to the platform important resiliency capabilities.

https://github.com/globaleaks/GlobaLeaks/issues/1658

**Comment by fpietrosanti** [comment]
I'm missing which is exactly the use case, what's the problem we are trying to satisfy, handling key escrow?

**Comment by evilaliv3** [comment]
No; the idea is to be able to archive all the data of a submission encrypted for an offline key.

but i agree that the cryptographic support for this implementation can be exactly the same that we can use for the key escrow mechanism that basically the same issue but applied to keys 

Also for the key escrow mechanism in fact we will need a with a public key associated to an offline private key.
